index.bs

```diff
@@ -424,6 +424,9 @@ following steps:
 
 </div>
 
+Isseue: TODO: Verify that [=domain-match=] clears all cookies for the etld+1
+    domain.
```

Comment:
To use the rigorously-defined term for etld+1. Also, domain-match itself seems like it just _finds_ the cookies, and that it's the next step that clears them.

```suggestion
Issue: TODO: Verify that using [=domain-match=] catches all cookies for the
    [=host/registrable domain=].
```